packages/core/docs/commercetools/use-facet.md

```diff
@@ -167,31 +167,67 @@ type ProductVariant = {
 }
 ```
 
+## Configuration
+
+::: warning
+Configuration can be changed only for th Enterprise version of this package.
+:::
+
+Faceting configuration can be modified to change available sorting options, filters, etc.
+
+If the explicit configuration is not provided, the following defaults will be used:
+
+```javascript
+{
+  pageOptions: [
+    20,
+    50,
+    100
+  ],
+  subcategoriesLimit: 100,
+  availableFacets: [
+    { facet: 'categories.id', type: 'string', option: 'subtree("*")', name: 'category' },
+    { facet: 'variants.attributes.size', type: 'number', option: '', name: 'size' },
+    { facet: 'variants.attributes.color.key', type: 'string', option: '', name: 'color' }
+  ],
+  sortingOptions: [
+    { id: 'latest', name: 'Latest', facet: 'createdAt', direction: 'desc' },
+    { id: 'price-up', name: 'Price from low to high', facet: 'price', direction: 'asc' },
+    { id: 'price-down', name: 'Price from high to low', facet: 'price', direction: 'desc' },
+    { id: 'relevance', name: 'Relevance', facet: 'score', direction: 'desc' },
+  ],
+  filteringStrategy: 'filter'
+}
```

Comment:
this tells me nothing about those options, I don't know what they are built for. Please mention the following things:
- `pageOptions` - available options of amount of possible elements on the page
- `subcategoriesLimit` - don't remember that... :D 
- `availableFacets` - that are the available filters bu using CT facets, add ref to:https://docs.commercetools.com/api/projects/products-search#facets `option` is the option of aggregation (eg. subtree) and `name` means alias (please also point that name 'category' is constant, user shouldn't change it)
-  `sortingOptions` - similar to previous option: https://docs.commercetools.com/api/projects/products-search#sorting
- `filteringStrategy` - https://docs.commercetools.com/api/projects/products-search#filters possible options here: filter, query and facets (or they are named differently in the rest API sdk)